### ISSUE TITLE
Missing dependency in lib-auth

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#363]
**Package**
lib-auth

**Feature or Issue Description**
`bootstrap.sh` fails when building lib-auth with a 404 error for one of its dependencies.
```
npm ERR! code E404
npm ERR! 404 Not Found: har-validator@5.1.2
```

**To Dos**

- [ ] Example

**Comment by shaunanoordin** [comment]
Update following our debug session...

**Problem**
- One of `lib-auth`'s dependencies uses `har-validator@5.1.2`
- `har-validator` DOESN'T have version 5.1.2 - see https://www.npmjs.com/package/har-validator

**Solution (well, _workaround,_ really)**
- Install the next available version: v5.1.3
- `$ cd packages/lib-auth`
- `$ cd npm install har-validator@5.1.3`
- `$ lerna bootstrap --scope=@zooniverse/auth`

**Comment by eatyourgreens** [comment]
`request` in lib-auth depends on `har-validator@~5.1.0` so deleting the package lock from lib-auth and rebuilding ought to fix this, in theory.
